packages/utils/util.ts

```diff
@@ -257,3 +257,14 @@ export const refAttacher = <T extends HTMLElement | ComponentPublicInstance>(
     ref.value = val
   }
 }
+
+export function createLink(link: string) {
+  const href: HTMLElement = document.createElement('a')
```

Comment:
Prefer `link` or `$a` for the variable name, href represents `hyper reference` which does not meet the context here.